packages/typespec-ts/test/modularUnit/scenarios/samples/parameters/parameterTypesCheck.md

```diff
@@ -0,0 +1,96 @@
+# Should generate samples for different types
+
+Sample generation should handle different types in body.
+
+## TypeSpec
+
+```tsp
+model Foo {
+  bar: string;
+  barDate?: utcDateTime;
+}
+
+model Widget {
+  strValue: string;
+  numValue: int32;
+  enumValue: "red" | "blue";
+  modelValue: Foo;
+  dateValue: utcDateTime;
+  arrValue: string[];
+  unionValue: Foo | string;
+  nullValue: null;
+  @clientName("jsClientName", "javascript")
+  renamedProp: string;
```

Comment:
Can you add number/boolean/string literal type?

Reply:
done.